### ISSUE TITLE
Something doesn't seem right with Info-nodes

### ISSUE THREAD

**Issue by agzam** [hlissner/doom-emacs#5745]
### What did you expect to happen?

In `Info-mode`, when you're in the Top node, using Go To Node `Info-goto-node` doesn't show any nodes, only this:

```
*
Top This is the top of the INFO tree
```

It makes it unable to navigate to selected nodes.

Trying to type `( Tab` and then a search for a topic shows some files, but the completion (tested with Vertico) is broken. It still won't allow you to search for a topic.

It doesn't seem like `Info-directory-list` get anyhow modified, anyone has any ideas?

### What actually happened?

Info-goto-node not working

### Steps to reproduce

1. Open Emacs
2. `M-x info`
3. Note that the Node indicator at the top should say: "Node: Top"
4. `M-x Info-goto-node`
5. Try to jump for example to `(elisp)` node

### System Information

https://pastebin.com/H7H6KWmD

**Comment by jkroes** [comment]
I'm having the same issue.

**Comment by condy0919** [comment]
Perhaps you need `M-x Info-menu` (`gm` when evil is enabled) rather than `M-x Info-goto-node` (`gG` when evil is enabled).

**Comment by agzam** [comment]
@condy0919 No, these are two different operations. One is to open a given Info manual, another is to jump within a manual. I ended up adding info+ package and binding a key to `info-display-manual` provided in the package.

**Comment by agzam** [comment]
@jkroes Ah, I see what's happening. Unfortunately, I am unable to select a topic. Vertico complains: `Error in post-comman-hook (vertico--exhibit) (wrong-number-of-arguments)....`

I'll try to search upstream, maybe someone has reported it there.

**Comment by minad** [comment]
@agzam There is nothing wrong with Vertico in my tests. Please compare how the default completion UI behaves and compare the behavior with Vertico.

The issues being discussed here are the following:

1. Duplicate candidates. This is minad/vertico#69 and one could indeed consider this a Vertico issue. For now I avoided to filter out duplicate candidates in Vertico since I consider the presence of duplicate candidates a bug in the backend. By not filtering I wanted to see which tables are actually affected by this, and as it turns out there are only a very small number of these buggy tables. Filtering in the frontend is of course not a big problem, except that it leads to unnecessary allocations and a small performance degradation in the frontend (at least as long as hashtable based delete-dups is used). As a compromise I could filter out consecutive duplicates, which does not incur a performance penalty.
2. The post-command-hook error is probably an upstream issue. I have not observed this before on Emacs 27.1. Maybe the bug has been introduced in a change in Emacs 28. You may want to submit a bug report to emacs-devel soon, such that this can be fixed before Emacs 28 is released. Note that you should be able to reproduce the bug with the default completion UI. The only difference is that you have to press TAB after entering a letter in order to trigger the bug.

**Comment by saolaolsson** [comment]
Not specifically related to this issue, but I got the same error message and traced it back to the [code from the vertico github wiki](https://github.com/minad/vertico/wiki#useful-commands-from-outside-minibuffer) which caused the error message in vertico 0.15 and 0.16 (but not 0.14) in Emacs 27.1.
